pegasus/sites.v3/code.org/public/about/index.haml

```diff
@@ -4,6 +4,7 @@ nav: about_nav
 video_player: true
 theme: responsive
 style_min: true
+set_dir: true
```

Comment:
Jorge requested the code.org/about page to also work for RTL languages, so I added this here even though this page has not been updated.